scene_update_rate: 60.0
render_rate: 30.0
scene_update_plugin:
  type: "gl_depth_sim::ROSURDFSceneUpdaterPlugin"
  params: ["no_params"]
render_plugins:
  type: "gl_depth_sim::LaserScannerPlugin"
  params:
    topic: "cloud"
    fixed_frame: "world"
    camera_frame: "camera"
    min_range: 0.01
    max_range: 10.0
    angular_resolution: 0.001
